src/components/RecipeView.vue

```diff
@@ -379,6 +379,10 @@ aside {
         margin-top: 2rem;
     }
 
+    .nutrition-items {
+        list-style-type: none !important;
```

Comment:
Is this really needed? I am always a bit suspicious when `!important` is in use.

Reply:
The other ways would be to either 

1. change the definition of `#app-content-wrapper aside ul` in the main `style.css` since this has a higher specificity than using the class
2. use `#app-content-wrapper aside ul.nutrition-items` which might be just as bad as using `!important` :D

For 1. I need to test if there are any unintended consequences when removing the definition. But changing it there might be a sensible way to handle this.

Reply:
See #515. The global CSS is a bad idea to change the styling of things within vue.

Reply:
For now, I suggest keeping the `!important` and opening an issue to remind us to come back to it eventually once #515 is finished.

Reply:
I forgot to push the updated version. We can start deprecating the global style here rather than introducing new code "depending" on the old style file which needs to be changed in the future.